.github/workflows/gating.yaml

```diff
@@ -95,3 +95,19 @@ jobs:
         pip install tox
     - name: Test with tox
       run: tox -e black
+
+  hadolint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        dockerfile:
+          - Dockerfile-api
+          - Dockerfile-flower
+          - Dockerfile-workers
+    steps:
+    - uses: actions/checkout@v2
+    - uses: hadolint/hadolint-action@v1.5.0
+      with:
+        dockerfile: docker/${{ matrix.dockerfile }}
+        ignore: DL3041
```

Comment:
I am not sure I know what is it, could you please add a comment?

Reply:
https://github.com/hadolint/hadolint/wiki/DL3041

Reply:
I added comment with detailed explanation about what is ignored